app/models/crop.rb

```diff
@@ -21,16 +22,18 @@ class Crop < ActiveRecord::Base
   before_destroy {|crop| crop.posts.clear}
 
   default_scope { order("lower(name) asc") }
```

Comment:
Can we put the where(:approval_status => "approved") into the default scope and save ourselves having to repeat it many times below?  Then when we want the pending approval ones, we just ask for those.